components/posts/postsList.js

```diff
@@ -33,11 +33,11 @@ export default function PostsList({ postData, mixpanel }) {
           >
             <Link href={"/" + post.slug}>
               <div className="flex flex-row space-x-2 items-center text-[1rem] md:text-[1.03rem] leading-5 tracking-wide capitalize">
-                <div className="relative w-[30px] h-[30px] md:w-[35px] md:h-[35px]">
+                <div className="relative w-[30px] md:w-[31px] h-[30px] max-w-full max-h-full overflow-hidden">
```

Comment:
Why need to take 31px in md devices?

Reply:
because [30px] had cut image on right side in md < devide.